tap/extensions/kafka/matcher.go

```diff
@@ -5,20 +5,19 @@ import (
 	"time"
 )
 
-var reqResMatcher = CreateResponseRequestMatcher() // global
 const maxTry int = 3000
 
 type RequestResponsePair struct {
 	Request  Request
 	Response Response
 }
 
-// Key is {client_addr}:{client_port}->{dest_addr}:{dest_port}::{correlation_id}
+// Key is {client_addr}:{client_port}_{dest_addr}:{dest_port}_{correlation_id}
```

Comment:
```suggestion
// Key is {client_addr_{client_port}_{dest_addr}_{dest_port}_{correlation_id}
```

Reply:
I think there is another place (not changed in this PR so I couldn't suggest an edit) that has comment with the old format key "with ":" and not "_". not sure it is super important bit if we already have some comment is should reflect the right explanation IMO

Reply:
@gadotroee oh yeah I missed this comment. Thanks!